doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -34,11 +34,11 @@ This Reference Architecture (RA-1) aims to provide an OpenStack distribution agn
 
 This reference architecture provides optionality in terms of pluggable components such as SDN, hardware acceleration and support tools.
 
-The Cloud Infrastructure layer includes the physical infrastructure which is then separated into virtual resources via a hypervisor.
-The VIM is expected to be OpenStack in line with the OpenStack Foundation core release.
+The Cloud Infrastructure layer includes the physical infrastructure which is then offered as virtual resources via a hypervisor.
+The VIM is expected to be OpenStack in line with the OpenInfra Foundation Train release.
```

Comment:
OpenInfra foundation does not have releases. Also I'm not even sure about the meaning of the original text. Wasn't the intention to specify that an OpenStack release what is still supported by the community should be used?

Reply:
@CsatariGergely We decided to just mention that this document is for the Train release. As you said, "the meaning of the original text" was unclear. I think the intent of Anuket only specifying OpenStack releases still supported by the community (and not vendors) probably belongs somewhere else, such as, in README.

Reply:
@CsatariGergely Changed sentence - please review. Thanks

Reply:
It stil reads like there would be a Train release of OpenInfra Foudation. I would propose simply "The VIM is expected to be OpenStack Train release."

Reply:
@CsatariGergely Fixed -- can you re-review? Thanks

Reply:
@CsatariGergely Can you please re-review? Thanks

@walterkozlowski can we merge if no response from @CsatariGergely since it has been quite a few days since fix? We are unable to add content for Lakelse until these Typo's are fixed because they affect the Kali release also. Thanks